### DISCUSSION TITLE
Add `nmi` (nautical miles) as valid distance unit

### DISCUSSION THREAD

**Discussion by mib1185** [home-assistant/architecture#1129] [Entity Models] [answered]
We already have knots (kn) as valid speed unit, which is just nmi per hour, we should also support nmi (nautical miles) as valid distance unit.

reference: https://community.home-assistant.io/t/add-nautical-miles-as-a-valid-unit-for-distance-device-class/754284

poc: https://github.com/home-assistant/core/pull/124723

**Comment by joostlek** [comment]
What core integrations can make use of this?

**Reply by mib1185** [reply]
No actual existing one, but there is a feature request about it (_see referenced forum post_), further I'm working on an [AIS tracker](https://github.com/home-assistant/core/pull/123956) integration, which aims to track vessels  (_and AIS is also the base for the users feature request_)

**Comment by frenck** [comment, marked as answer]
✅  We've discussed this proposal in the core meeting, and approved this one.

../Frenck

**Reply by frenck** [reply]
Shipped in https://github.com/home-assistant/core/pull/124723
